modules/light-clients/08-wasm/keeper/msg_server_test.go

```diff
@@ -62,7 +62,7 @@ func (suite *KeeperTestSuite) TestMsgStoreCode() {
 			func() {
 				msg = types.NewMsgStoreCode(signer, []byte{0, 1, 3, 4})
 			},
-			errors.New("Wasm bytes do not not start with Wasm magic number"),
```

Comment:
somebody already made this change and I had to revert it as it broke tests and it is outside of our scope - since the error is propagated from wasmvm.

I delivered a PR there to fix it https://github.com/CosmWasm/wasmvm/pull/531 but it hasn't been included in v2.0.1

If you could please revert this change we'd be happy to accept the PR